README.md

```diff
@@ -53,6 +53,58 @@ bazel build node_generator:joshua_main
 
 TODO: Add calibration instruction here.
 
+## Troubleshooting
+
+### If your robot is not moving, check Serial Device Permission Issues (`/dev/ttyACM0`)
+
+#### 1. Check Device Existence and Permissions
+```bash
+ls -la /dev/ttyACM*
+```
+**Expected output**: `crw-rw---- 1 root dialout 166, 0 /dev/ttyACM0`
+
+If you encounter permission errors when accessing serial devices like `/dev/ttyACM0`, follow these diagnostic steps:
+
+#### 2. Verify Your User Groups
+```bash
+groups $USER
+```
+**Check if**: `dialout` group is listed in your groups
+
+#### 3. Test Device Access
+```bash
+timeout 2 cat /dev/ttyACM0 2>&1 || echo "Cannot read from device"
+```
+
+#### 4. Diagnose the Issue
+- **If device doesn't exist**: Check hardware connection and power
+- **If permission denied**: You need `dialout` group membership
+- **If device exists but no permission**: Add yourself to dialout group
+
+#### 5. Fix Permission Issues
+```bash
+# Add your user to the dialout group
+sudo usermod -a -G dialout $USER
```

Comment:
I already add this in the main readme.
`sudo usermod -aG dialout $USER # Then reboot`

It doesn't fix the issue or you just missed setup?